File: Practice/21/Python/task21.py
def bmi(weight, height):
	return (weight / (height ** 2))

def print_bmi(bmi):
	if (bmi < 18.5):
		print("недостаточная масса")
	elif (bmi < 25.0):
		print("норма")
	elif (bmi < 30):
		print("избыточная масса")
	else:
		print("ожирение")

w,h = input().split()
w = float(w)
h = float(h) / 100
print_bmi(bmi(w, h))
input("Нажмите Enter")

File: Practice/21/C++/task21/task21.cpp
#include <iostream>

using namespace std;

double BMI(double weight, double height)
{
	return (weight / (height * height));
}

void printBMI(double BMI)
{
	if (BMI < 18.5) {
		cout << "Недостаточная масса";
	}
	else {
		if (BMI < 25.0) {
			cout << "Норма";
		}
		else {
			if (BMI < 30.0) {
				cout << "Избыточная масса";
			}
			else {
				cout << "Ожирение";
			}
		}
	}
}

int main()
{
	double w, h;
	setlocale(LC_ALL, "Russian");
	cin >> w >> h;
	h /= 100;
	printBMI(BMI(w, h));
	return 0;
}
